src/twisted/web/test/test_web.py

```diff
@@ -1849,3 +1852,39 @@ def test_defaultReactor(self):
 
         factory = http.HTTPFactory()
         self.assertIs(factory.reactor, reactor)
+
+
+class QueueResource(Resource):
+    isLeaf = True
+
+    def __init__(self) -> None:
+        super().__init__()
+        self.queue: List[Request] = []
```

Comment:
I had to read the test more than once to understand how the tests is designed.

Maybe this is a better name.

```suggestion
        self.dispatchedRequests: List[Request] = []
```